samples/twilio/program.py

```diff
@@ -33,22 +33,25 @@ def on_http_get(event):
     parameter 'to'. The message will be sent both as an SMS and a WhatsApp
     message to the specified number.
 
+    Note: No need to include the `+` prefix in the `to` parameter.
+    The program automatically appends the `+` where required.
+
     Args:
         event (object): An event object containing the request data.
     """
-    to = event.data["url"]["query"]["to"]
+    to = event.data["url"]["query"]["to"].strip()
     # Send SMS text via Twilio
     message = t.messages.create(
         from_=FROM_PHONE_NUMBER,
-        to=to,
+        to="+" + to,
```

Comment:
The thing is that even if the user adds "+" it won't matter because when we get it from the event we will get only the numbers. because of that reason we decided to add the "+", so even if the user types + or not this code will work

Reply:
@pashafateev ack on the motivation

@mario99logic it depends - when the "+" is URL-encoded (manually by the user, or automatically by curl/wget/etc.) then it will be received correctly - so see my alternative suggestion above